pkg/helper/bridges_test.go

```diff
@@ -126,7 +126,13 @@ var _ = Describe("Network desired state bridge parser", func() {
 		})
 		It("should return the slice with the bridges", func() {
 			Expect(err).ToNot(HaveOccurred())
-			Expect(obtainedBridges).To(ConsistOf("br1", "br2"))
+			Expect(len(obtainedBridges)).To(Equal(2))
```

Comment:
Can we add a test with some bridges down and also son bridges up but without ports ?

Reply:
I'm all for more tests, but I'd rather have those in separate follow-up PRs (like this https://github.com/nmstate/kubernetes-nmstate/pull/160). We should get basic fix merged ASAP to be able to test it on real environments.